qiskit_machine_learning/neural_networks/__init__.py

```diff
@@ -57,6 +57,15 @@
    EffectiveDimension
    LocalEffectiveDimension
 
+Neural Networks using Primitives
```

Comment:
Do we want to separate them out like this? The existing networks will be marked Pending Deprecation and we normally change the first sentence that comes out in the resultant html table here to prefix it with Pending Deprecation so you would see at a glance here some are pending deprecation.

Reply:
Removed. I think that we will leave the deprecation warnings for a follow-up PR